config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/opdev/l5-operator
+  newName: quay.io/skattoju/opdev/l5-operator
```

Comment:
Is that expected?

Reply:
nope bug had some extraneous environment variables set